README.md

```diff
@@ -1,38 +1,123 @@
 
-1. Make a mutation by going offline.
-2. Then, come online and click on "FromDb" button and you will see the results.
-3. Also, refresh the application to see the updated cache.
+ **Consuming Code**
 
+  **Code for performing Mutations** 
+  
+  ```groovy
+    //Create an Object of mutation by passing in the build parameters according to the scehma.
+    val mutation = UpdateCurrentTaskMutation.builder().id(id).title(title).version(version).build()
+    
+    //Create a client which is an object of ApolloCall on which call would be made.
+     val client = Utils.getApolloClient(context)?.mutate(mutation)?.refetchQueries(apolloQueryWatcher?.operation()?.name())
+     
+    //Create a callback object of type ApolloCall.Callback<UpdateCurrentTaskMutation.Data>
+     val callback = object : ApolloCall.Callback<UpdateCurrentTaskMutation.Data>() {
+            override fun onFailure(e: ApolloException) {
+                e.printStackTrace()
+            }
+
+            override fun onResponse(response: Response<UpdateCurrentTaskMutation.Data>) {
+              //Perform UI Bindings here.
+            }
+        }
+        
+     /*Call the enqueue function on the instance of APolloClient and pass in two parameters here:
+       1. mutation object typecasted as mutation as 
+          com.apollographql.apollo.api.Mutation<Operation.Data, Any, Operation.Variables>                 
+       2. callback object typecasted as callback as ApolloCall.Callback<Any>
+     */  
+     Utils.getApolloClient(context)?.enqueue(
+            mutation as com.apollographql.apollo.api.Mutation<Operation.Data, Any, Operation.Variables>,
+            callback as ApolloCall.Callback<Any>
+        )
+```
+
+## Run the sample app
+
+- Run the application to send query and mutation to the server and displaying the results to the user.
+
+### Display of Offline Capabilities 
+
+1. Make any mutation by going offline.
+2. Then, after you come online your mutations (made when you are offline) will hit the server and you will get the response      back from the server.
+3. To get the most recent data in your application, **refresh** your app once after the network comes back by swiping down on    the screen.
 
 ## Demo 
 
-Here the client performs a mutation of updating the Task3 when it is offline and when it comes online back again, changes are refelected back in the application on clicking on the "FromDB" button present on the top of the screen.
+![OfflineFore1gif](https://user-images.githubusercontent.com/33238323/61216474-1177b180-a72b-11e9-883a-8592d09ee290.gif)
+
+
+## Bugs
+
+Offline implementation works only when the app is not closed. If you close the app, the mutationns done while you are offline are not persisted yet. 
```

Comment:
Rephrase this, 
1. Offline mutations are lost when the app is closed.
2. No UI Bindings, you will have to update your UI manually.